components/product/ProductCard.js

```diff
@@ -23,6 +24,20 @@ function ProductCard({ product, store, navigation }) {
           style={{ height: 86, width: 86, borderRadius: 12 }}
         />
         <Body>{product.name}</Body>
+        {product.detail && (
+          <Caption color={Colors.secondaryText}>{product.detail}</Caption>
+        )}
+        <RowContainer>
+          <Caption color={Colors.secondaryText}>
+            ${product.customerCost.toFixed(2)} ea
```

Comment:
Could we use the `displayDollarValue` here? Not sure if that's right since it's in `RewardsUtils`, but just a thought.

Reply:
Hmm maybe we can move it to `common(Utils).js` in this PR? Either way I think that's fine